src/index.html

```diff
@@ -6,9 +6,68 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logo">
+        <a href="index.html">
+          <img
+            alt="logo"
+            src="images/logo.png"
+            class="logo__img"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item ">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover">
+              Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/df40cf57-6b0a-4318-8d4c-18e9cc609b16)